torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -955,6 +955,21 @@ void TensorExprKernel::LowerToBackend(BackendType backend_type) {
             "Invalid loop-level: " + std::to_string(loop_levels));
       }
     }
+  } else if (backend_type == kLLVMCodeGen) {
```

Comment:
Could you post some benchmarks numbers to make sure this is reasonable?

Feel free to use the one under bendchmarks/tensorexpr as a starting point.

Reply:
Here's benchmark results:  https://hastebin.com/raw/datefugisi

That said, I don't think we should be reading into these too much.  Any place we're doing well is accidental rather than intentional, so there's going to be a lot of up/down noise as we move from no-heuristics to some-heuristics.